src/main/java/com/pru/offboarding/task/RunAfterStartUp.java

```diff
@@ -0,0 +1,100 @@
+package com.pru.offboarding.task;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+
+import com.pru.offboarding.task.entity.OffBoardingChecklist;
+import com.pru.offboarding.task.entity.Recording;
+import com.pru.offboarding.task.entity.Training;
+import com.pru.offboarding.task.repo.OnBoardingChecklistRepo;
+import com.pru.offboarding.task.repo.RecordingRepo;
+import com.pru.offboarding.task.repo.TrainingRepo;
+
+
+
+@Component
+public class RunAfterStartUp {
+	
+	@Autowired
+	private TrainingRepo trainingRepo;
+	
+	@Autowired
+	private RecordingRepo recordingRepo;
+	
+	@Autowired
+	private OnBoardingChecklistRepo onBoardingChecklistRepo;
+	
+	@EventListener(ApplicationReadyEvent.class)
+	public void runAfterStartup() {
```

Comment:
No hard-coded values please, once the application s up, the user needs to set up the users/meta-data/other required data from the application itself.